kolibri/plugins/learn/assets/src/views/LibraryPage/ResumableContentGrid.vue

```diff
@@ -88,24 +87,8 @@
       const { resumableContentNodes, moreResumableContentNodes, fetchMoreResumableContentNodes } =
         useLearnerResources();
 
-      var sidePanelContent = ref(null);
-      const toggleInfoPanel = content => (sidePanelContent.value = content);
-
-      const activityRefs = {};
-
-      const findFirstEl = () => {
-        const activityKeys = Object.keys(activityRefs);
-        const firstActivity = activityKeys.find(key => key.endsWith('0'));
-        if (firstActivity) {
-          activityKeys[firstActivity].value.focusFirstEl();
```

Comment:
This seems like something we may have updated to be better managed, so therefore not needed and just not cleaned up, but it also makes me wonder if we lost some accessibility stuff here due to a regression. I will see if I can trace back a bit and remember in the git history

Reply:
Yes, although I suspect this might have been copy pasta from elsewhere - as the reference to sidepanel content (when this does not use a side panel at all) and activityRefs made me suspect that this is coming from something not at all related to showing a grid of resumable content.

Reply:
yes, that's a good point